tools/segmetrics/segmetrics.xml

```diff
@@ -1,10 +1,14 @@
-<tool id="ip_segmetrics" name="Compute image segmentation and object detection performance measures" version="1.4.0-2" profile="20.05">
+<tool id="ip_segmetrics" name="Compute image segmentation and object detection performance measures" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.05">
     <description>with SegMetrics</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.4</token>

```

Comment:
I can not promise you that 1.4+galaxy is higher than 1.4.0-1 ... we risk it or try that out before. Need to find the snippet how Pyhton and Galaxy calculates the version.

Reply:
Since there are no changes besides the versioning scheme, we might as well remove the 1.4.0-1 version from toolshed, I guess?

Reply:
We can not remove stuff from the TS - reproducibility.

Reply:
Yes, this we can do.